static/sass/_pattern_navigation.scss

```diff
@@ -417,15 +420,15 @@ $meganav-height: 3rem;
       }
     }
 
-    button.p-navigation__link.js-menu-button {
-      padding-left: 1rem;
-      padding-right: 1rem;
-    }
-
     .p-navigation__item {
       .p-navigation__link {
         color: $color-mid-light;
       }
+
+      .js-menu-button.p-navigation__link {
+        padding-left: 1rem;
```

Comment:
At least for bubbles I think you need to remove the left padding on the first item, otherwise it won't be aligned to the hero text:
![Screenshot from 2024-01-04 15-26-17](https://github.com/canonical/ubuntu.com/assets/17607612/b28bab2b-b25e-46bc-a483-5b53f455039f)

Reply:
I wasn't sure about this, as we would need to also align it to the navigation about (so 'your subscriptions' and 'products'). As well, assuming we are just aligning the text inside the button itself, it might look a bit weird on hover, or when it is the active page. something like this (It wont happen on this page as 'your subscriptions' doesn't keep you in the bubble):
![image](https://github.com/canonical/ubuntu.com/assets/58276363/d817ad68-b633-4ab0-b720-a938f493fe6d)
What are you thoughts on this @juanruitina?

Reply:
We had the same problem on c.com fwiw. I don't remember what we ended up doing, thought it was removing the left padding but it doesn't look like that's right. Maybe taking a look at that repo would be helpful. This is what we ended up with for example, same thing for bubbles like /data: 
![Screenshot from 2024-01-05 12-03-55](https://github.com/canonical/ubuntu.com/assets/17607612/4a6f098b-be49-428e-b01c-f83a681ceb32)
![Screenshot from 2024-01-05 12-04-05](https://github.com/canonical/ubuntu.com/assets/17607612/df97d8e0-5d9b-4fc5-98ac-fecb5e01c874)

Reply:
I see in canonical.com that we added some negative margin: `p-navigation__items` has `margin-left: -0.75rem;` This way the hover/active states look OK: it is the text that needs to align, not so much the underlining. (Demo is failing so I couldn't manage to test it :/).